bin/sirius-script-rabpm-mngr.py

```diff
@@ -61,8 +61,28 @@ def run_boot_from_flash(rack, slot):
 
 
 def add_rack_arg(command):
-    command.add_argument("-r", "--rack", required=True,
-                         help="Rack number {01..20} or 21 for transport line")
+    command.add_argument("-r", "--racks", required=True,
+                         nargs="+", action=ValidateDeviceList, all_devices=list(range(1, 22)),
+                         help="Rack numbers {01..21} (21 for transport line) or 'all'")
+
+
+class ValidateDeviceList(argparse.Action):
+    def __init__(self, option_strings, all_devices, *args, **kwargs):
+        self._all_devices = all_devices
+        super(ValidateDeviceList, self).__init__(
+            option_strings=option_strings, *args, **kwargs)
+
+    def __call__(self, parser, args, values, option_string=None):
```

Comment:
I'd keep the same argument names from the [superclass](https://docs.python.org/3/library/argparse.html#argparse.Action.__call__), which means replacing `args` with `namespace`.